### ISSUE TITLE
Toggle random play controls switched

### ISSUE THREAD

**Issue by dradice** [TheStalwart/Theremin#15]
The controls for switching the random play on/off seems a bit counter intuitive to me: in order to switch off random play one needs to click an icon with two arrows pointing in opposite directions, while, to switch on random play, one needs to click an icon with two arrows pointing in the same directions. It seems to me that the opposite convention would be more useful. This would also be consistent with the behavior of the play/pause button.

**Comment by TheStalwart** [comment]
I find that button confusing too, and usually toggle random and repeat options from main menu. But the only way to make it unambiguous is to make it light up, like the one on iTunes, Winamp, and pretty much every other music player.

**Comment by dradice** [comment]
I agree. I created a pull request because I found no way to include a patch in the bug report and not because I think that this new arrangement is necessarily better: it is a bit more clear for me, but it might be more confusing for others (especially for old users used to the old arrangement)... I just wanted to leave a simple workaround for people, like me that prefer the other arrangement.

I might look into the iTunes-like solution in the future, but honestly there are few chances that I will find time for it... I have almost zero experience with GUI programming and absolutely zero experience with Objective C...

**Comment by TheStalwart** [comment]
Theremin's codebase is terribly bitrotten anyway. I started a rewrite few months ago, but got carried away by other hobby projects.

**Comment by OogieM** [comment]
I'm new to Mac programming but very interested as well. I'm doing Java though, my last C programming was 30years ago and it sure wasn't Objective C! FWIW I found that random button behavior confusing as well and look forward to the change.
